### DISCUSSION TITLE
Gradle Java client library Android Studio issue

### DISCUSSION THREAD

**Discussion by SyedAbbas01** [ory/kratos#1148] [Q&A] [answered]
Trying to follow the guide here https://github.com/ory/hydra-client-java

Added here:

![image](https://user-images.githubusercontent.com/38494273/111084740-cf100480-850b-11eb-9de2-910f55cc1bb4.png)

But seems the file is not there to be downloaded:
![image](https://user-images.githubusercontent.com/38494273/111084774-041c5700-850c-11eb-8745-17223b3d7726.png)
![image](https://user-images.githubusercontent.com/38494273/111084809-2f9f4180-850c-11eb-879f-d346d567a6c0.png)

**Comment by aeneasr** [comment]
The version is not correct. Use a released version

**Reply by SyedAbbas01** [reply]
Can you clarify which one I need please:

![image](https://user-images.githubusercontent.com/38494273/111132243-082e9000-8571-11eb-9fd1-683e9f9aae11.png)

**Reply by vinckr** [reply]
Hey @SyedAbbas01, 
The client is autogenerated and has not yet been updated to 1.9.1. 
But you can use 1.9.0: https://search.maven.org/artifact/sh.ory.hydra/hydra-client/1.9.0/jar .

**Reply by SyedAbbas01** [reply]
Thanks Vincent I added the wrong link I meant for Kratos

have found the one that works:

https://search.maven.org/artifact/sh.ory.kratos/kratos-client/0.5.4-alpha.1/jar

Dependancy:

 implementation 'sh.ory.kratos:kratos-client:0.5.4-alpha.1'

To avoid duplicate class name error:
    configurations.all {exclude group: 'org.apache.oltu.oauth2', module: 'org.apache.oltu.oauth2.common'}
    
It is building but when I run it I get this error:

![image](https://user-images.githubusercontent.com/38494273/111151252-1e474b00-8587-11eb-8286-9377f4886424.png)

**Reply by vinckr** [reply]
Ah ok. 
I havent used this client yet, but I can try to check it out sometime this week.

**Comment by vinckr** [comment, marked as answer]
Hey, @SyedAbbas01 sorry for the late follow-up!
The client has since been updated: https://search.maven.org/artifact/sh.ory.kratos/kratos-client/0.6.3-alpha.1/jar
Let me know if anything is still not working as expected.
